src/Sylius/Bundle/ApiBundle/Resources/config/routing/main.yml

```diff
@@ -20,13 +20,22 @@ sylius_api_taxon:
     resource: "@SyliusApiBundle/Resources/config/routing/taxon.yml"
     prefix: /taxons
 
+sylius_api_cart:
+    resource: "@SyliusApiBundle/Resources/config/routing/cart.yml"
+    prefix: /carts
+
 sylius_api_order:
-    resource: "@SyliusApiBundle/Resources/config/routing/order.yml"
-    prefix: /orders
+    resource: |
+        alias: sylius.order
+        section: api
+        only: [show, delete]
```

Comment:
IMO we should not allow for deleting orders, you can only cancel already created ones.